docs/reference/executionModes.md

```diff
@@ -258,22 +263,7 @@ This can be achieved by using FileIncrementalMoveMode. If option `archiveSubdire
 
 FileIncrementalMoveMode can be used with the file engine (see also [Execution engines](executionEngines.md)), but also with SparkFileDataObjects and the data frame engine.
 
-## Others
-
-### ProcessAllMode
-An execution mode which forces processing of all data from its inputs.
-Any partitionValues and filter conditions received from previous actions are ignored.
-
-### CustomPartitionMode
-This execution mode allows to implement arbitrary custom partition logic using Scala.
-
-Implement trait `CustomPartitionMode` by defining a function which receives the main input and output DataObjects
-and returns the partition values that need to be processed.
 
-### CustomMode
```

Comment:
CustomMode is not mentioned above. We should keep this text.

Reply:
done